src/style/star.css

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+  
+.stars__star {
+  background: url(../images/star-active.svg)  no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  align-items: center;
+}
+    
+.stars--4 .stars__star:nth-child(n+5) {
```

Comment:
Please also add cases for different count of stars

Reply:
i did